apps/test/unit/code-studio/peer_reviews/PeerReviewSubmissionsTest.jsx

```diff
@@ -85,8 +85,9 @@ describe('PeerReviewSubmissions', () => {
       submissions: fakePeerReviewData.submissions,
       pagination: fakePeerReviewData.pagination
     });
-    expect(peerReviewSubmissions.find('#DownloadCsvReport').prop('disabled')).to
-      .be.true;
+    expect(
+      peerReviewSubmissions.find('button#DownloadCsvReport').prop('disabled')
+    ).to.be.true;
```

Comment:
Had to adjust selectors throughout this test because [`find()` now returns host nodes and DOM nodes](https://github.com/airbnb/enzyme/blob/master/docs/guides/migration-from-2-to-3.md#find-now-returns-host-nodes-and-dom-nodes)